packages/api-explorer/src/scenes/MethodScene/MethodScene.tsx

```diff
@@ -108,8 +108,14 @@ export const MethodScene: FC<DocMethodProps> = ({ api }) => {
         <Aside width="50rem">
           <ComponentsProvider
```

Comment:
We actually built `ExtendComponentsProvider` for this exact use case.

```suggestion
          <ExtendComponentsThemeProvider
```